commands/types/fakedoor/message.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var messageFmt = "The support for %s hasn't been implemented yet.\nIf you would like us to implement it, please drop by at:\n%v and add thumbs up.\nWe're happy to implement it quickly based on demand!"
```

Comment:
How about adding an 👍? 😄

Reply:
If you are not planning on modifying this for a test, "const" can be more suitable

Reply:
> How about adding an 👍? 😄

I have added `thumbs up sign` as a unicode at the end of the message. I guess sign as a unicode is universally compatible for each system, even though there is exception such as `powershell`.

> If you are not planning on modifying this for a test, "const" can be more suitable

I guess, fake door context is narrow enough to give up a test. So I think we can change it as const.
Changed [6f49af6](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/6f49af6098d1a9729952ba84f745aa75ae657b09).